.github/workflows/composer-require-checker.yml

```diff
@@ -48,6 +50,7 @@ jobs:
           php-version: ${{ matrix.php }}
           extensions: ${{ inputs.extensions }}
           coverage: none
+          tools: composer-require-checker
```

Comment:
I think we're currently installing it via composer plugin. Should we remove it from there?

Reply:
I would like us to just run it in ci, basically the IDE gives us the dependencies, and if it fails in ci, the developer can fix it, and we don't have to add more dependencies, which for me is a plus.